packages/flame_oxygen/example/lib/system/debug_system.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flame/game.dart';
+import 'package:flame_oxygen/flame_oxygen.dart';
+import 'package:flutter/material.dart';
+
+class DebugSystem extends BaseSystem {
+  final debugPaint = Paint()
+    ..color = Colors.green
+    ..style = PaintingStyle.stroke;
+
+  final textPainter = TextPaint(
+    config: const TextPaintConfig(
+      color: Colors.green,
+      fontSize: 10,
+    ),
+  );
+
+  @override
+  List<Filter<Component>> get filters => [];
+
+  @override
+  void render(Canvas canvas) {
+    super.render(canvas);
+    textPainter.copyWith((config) => config.withFontSize(16)).render(
```

Comment:
It shouldn't need to make a new `TextPaint` in each render tick right?

Reply:
Did this to have an easy way to render the same kind of text but bigger, but I reworked it now